packages/manager/src/components/LandingHeader/LandingHeader.tsx

```diff
@@ -108,17 +112,19 @@ export const LandingHeader = ({
               flex: '1 1 auto',
 
               marginLeft: customSmMdBetweenBreakpoint
-                ? theme.spacing(2)
+                ? theme.spacingFunction(16)
                 : customXsDownBreakpoint
-                  ? theme.spacing(1)
+                  ? theme.spacingFunction(8)
                   : undefined,
             }}
           >
             {betaFeedbackLink && (
               <span
                 style={{
-                  marginLeft: xsDown ? `${theme.spacing(2)}` : undefined,
-                  marginRight: `${theme.spacing(2)}`,
+                  marginLeft: xsDown
+                    ? `${theme.spacingFunction(16)}`
+                    : undefined,
+                  marginRight: `${theme.spacingFunction(16)}`,
```

Comment:
Nit: we don't need template literals here

```suggestion
                  marginLeft: xsDown
                    ? theme.spacingFunction(16)
                    : undefined,
                  marginRight: theme.spacingFunction(16),
```